packages/safe-tools-client/config/deploy.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env node */
+
+module.exports = function (deployTarget) {
+  let ENV = {
+    s3: {
+      allowOverwrite: true,
+    },
+    cloudfront: {
+      objectPaths: ['/*'],
+    },
+  };
+
+  if (deployTarget === 'staging') {
+    ENV.build.environment = 'production';
+    ENV.s3.bucket = 'cardstack-safe-tools-client-staging';
+    ENV.cloudfront.distribution = 'PLACEHOLDER';
```

Comment:
Do we know our distribution IDs and/or we make them available as environment variables to Github Actions?

Reply:
They’re [hardcoded](https://github.com/cardstack/cardstack/blob/main/packages/web-client/config/deploy.js#L111) for web-client (and ssr-web [previews](https://github.com/cardstack/cardstack/blob/main/packages/ssr-web/config/deploy.js#L77))